### PR TITLE
run some real tests against mongodb 2.6 in travis

### PR DESCRIPTION
**Pull request by vkarpov15**
@kyeah @saghmrossi this will enable us to run tests in Travis until we get insert working.

The only test that's excluded for now is the `list_collections()` one, because list_collections got a big overhaul in 3.0 and our current implementation is incompatible with mongodb 2.6, which is fine. We'll figure out a way to use [mongodb-version-manager](https://www.npmjs.com/package/mongodb-version-manager) or something similar to make travis use mongodb 3.0 sometime next week.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,6 +12,8 @@ before_install:
     - sudo make install
     - sudo ldconfig /usr/local/lib
 
+before_script: "echo \"db.test.insert({ foo: 42 }); db.getSiblingDB('sample').movies.insert({ title: 'Jaws' });\" | mongo"
```
Comment (saghmrossi) [inline .travis.yml]: What's the reasoning behind putting { foo: 42 } in the database before running the tests? Right now, the wire protocol tests assume that the database is empty initially.

Comment (vkarpov15) [inline .travis.yml]: I haven't adjusted this for your outstanding PR yet. The code in 1.0 right now assumes that there's a `{ foo: 42 }` document in 'test.test'

Comment (saghmrossi) [inline .travis.yml]: Ah, okay, makes sense

File: .travis.yml [hunk 2]
```diff
@@ -9,6 +12,8 @@ before_install:
     - sudo make install
     - sudo ldconfig /usr/local/lib
 
+before_script: "db.getSiblingDB('sample').movies.insert({ title: 'Jaws' });\" | mongo"
```
Comment (kyeah) [inline .travis.yml]: Missing `echo \"` before `db.getSiblingDB`, which is killing the travis build.

**Pull request closed without merging**